src/catalog/controller/checkout/pingback.php

```diff
@@ -24,7 +23,7 @@ public function index()
         $this->loadPaymentModel($order['payment_code']);
         $this->getPaymentModel()->initConfig(true);
 
-        $pingback = new Paymentwall_Pingback($request, $this->getRealIpAddress($this->request->server));
+        $pingback = new Paymentwall_Pingback($_GET, $this->getRealIpAddress($this->request->server));

```

Comment:
please use `$this->request->get` or `$this->request->request` instead of super global variables